android/app/src/main/AndroidManifest.xml

```diff
@@ -123,6 +123,12 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/file_provider_paths" />
         </provider>
+        <meta-data
+            android:name="com.google.firebase.messaging.default_notification_icon"
+            android:resource="@drawable/notification_icon" />
+	    <meta-data
+            android:name="com.google.firebase.messaging.default_notification_color"
+            android:resource="@color/notification_icon_color" tools:replace="android:resource" />
```

Comment:
Here we're updating the Android app manifest to define the notification color for the notification app icon.